### PR TITLE
ostinato: fix build

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
With libpcap 1.8.1 programs that define `HAVE_REMOTE` macro fail to build. See debian bug [report](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=843384). These include package ostinato (and probably others) which is needed for release 17.03 (issue #23253).

I added the debian patch to libcap and update ostinato.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using (still running)
- [x] Tested execution of all binary files
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/ostinato/default.nix [hunk 1]
```diff
@@ -22,42 +24,41 @@ stdenv.mkDerivation rec {
...
 
   meta = with stdenv.lib; {
     description = "A packet traffic generator and analyzer";
-    homepage = http://ostinato.org;
-    license = licenses.gpl3;
+    homepage    = http://ostinato.org;
+    license     = licenses.gpl3;
     maintainers = with maintainers; [ rick68 ];
-    platforms = platforms.linux;  # also OS X and cygwin
+    platforms   = with platforms; linux + darwin + cygwin;
```
Comment (dezgeg) [inline pkgs/applications/networking/ostinato/default.nix]: This needs to use `++` not `+`.

Comment (rnhmjoj) [inline pkgs/applications/networking/ostinato/default.nix]: Thank you

**Comment by dezgeg** [comment]
Next up:

````
output path ‘/nix/store/ia85wb28286hj4p2ra7bvz4164hhrdl3-disable-remote.diff’ has sha256 hash ‘0dvjax9c0spvq8cdjnkbnm65wlzaml259yragf95kzg611vszfmj’ when ‘1jby2yi1miyr3b5l63w014vccmliir0l15hba0nc9fzvs71xnryx’ was expected
````

Probably you put the hash of `fetchurl` to the expression that uses `fetchpatch`.

**Comment by rnhmjoj** [comment]
Thanks for the help.

**Pull request merged**